tests/Google/Ads/GoogleAds/V0/Util/GoogleAdsFailuresTest.php

```diff
@@ -15,70 +15,46 @@
  * limitations under the License.
  */
 
-namespace Google\Ads\GoogleAds\V0\Util;
+namespace Google\Ads\GoogleAds\Util\V0;
 
-use \Google\Protobuf\Any;
-use \Google\Rpc\Status;
+use Google\Protobuf\Any;
+use Google\Rpc\Status;
 use Google\Ads\GoogleAds\V0\Errors\GoogleAdsFailure;
 use PHPUnit\Framework\TestCase;
 
 class GoogleAdsFailureTest extends TestCase
 {
     public function testFromAny()
     {
-        $failureMock = $this
-            ->getMockBuilder(GoogleAdsFailure::class)
-            ->disableOriginalConstructor()
-            ->getMock();
-
         $anyMock = $this
             ->getMockBuilder(Any::class)
             ->disableOriginalConstructor()
             ->getMock();
         $anyMock
             ->method("unpack")
-            ->willReturn($failureMock);
+            ->willReturn(new GoogleAdsFailure());
         
         $this->assertInstanceOf(GoogleAdsFailure::class, GoogleAdsFailures::fromAny($anyMock));
     }
 
     public function testFromEmptyStatus()
     {
-        $emptyStatusMock = $this
-            ->getMockBuilder(Status::class)
-            ->disableOriginalConstructor()
-            ->getMock();
-        $emptyStatusMock
-            ->method("getDetails")
-            ->willReturn([]);
-
-        $this->assertCount(0, GoogleAdsFailures::fromStatus($emptyStatusMock));
+        $emptyStatus = new Status();
```

Comment:
Nit: this assignment wouldn't be necessary.

Reply:
Done.